docs/my-website/docs/observability/keywordsai_integration.md

```diff
@@ -0,0 +1,73 @@
+# 🗝️ Keywords AI - LLM monitoring platform
+
+:::tip
+
+This is community maintained. Please make an issue if you run into a bug:
+https://github.com/BerriAI/litellm
+
+:::
+
+[Keywords AI](https://keywordsai.co/) makes it easy for developers to build LLM applications. With 2 lines of code, developers get a complete monitoring platform that speeds up deploying & monitoring AI apps in production.
+
+## Using Keywords AI with LiteLLM
+
+LiteLLM provides `callbacks`, allowing you to easily log data to Keywords AI based on the status of your responses.
+
+### Supported LLM Providers
+
+Keywords AI can log requests across [various LLM providers](https://docs.keywordsai.co/integration/supported-models)
+
+### Integration Methods
+
+There are two main approaches to integrate Keywords AI with LiteLLM:
+
+1. Using callbacks
+2. Using Keywords AI as a proxy
+
+### Approach 1: Use Callbacks
+
+Use just 2 line of code to log your responses with Keywords AI:
+
+```python
+from litellm.integrations.keywordsai import KeywordsAILogger
+litellm.callbacks = [KeywordsAILogger]
```

Comment:
please fix this to work as expected `litellm.callbacks = ["keywords_ai"]`

Reply:
Changed